tools/DebianPackageTool/package_tool

```diff
@@ -152,6 +152,9 @@ package_all(){
     package_absolute_placement
     package_samples
     package_docs
+    if [[ $PACKAGE_NAME == "dotnet-dev-"* ]]; then
+        package_debian
```

Comment:
I didn't initially have the guard but then it ended up trying to copy postinst into the sharedframework package which failed. Was being a little quick and dirty to get something working. Open to whatever the right thing to do is.

Agree the name sucks.